k8s-daemonset/k8s/hello-world-latency.yml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: hello
+spec:
+  replicas: 1
```

Comment:
Mind setting this to `replicas: 3`? I think that's a better example to set. Same goes for the world RC in this file.